staging/src/k8s.io/sample-controller/controller.go

```diff
@@ -196,50 +198,50 @@ func (c *Controller) runWorker(ctx context.Context) {
 // processNextWorkItem will read a single work item off the workqueue and
 // attempt to process it, by calling the syncHandler.
 func (c *Controller) processNextWorkItem(ctx context.Context) bool {
```

Comment:
This aligns to the example in our docs: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/controllers.md .  there's been a little drift over time and we should collapse the discrepancy.

Reply:
@deads2k , I am not sure I understand your point. Which are you suggesting should change, and how?